deploy-queue/Cargo.toml

```diff
@@ -3,6 +3,14 @@ name = "deploy-queue"
 version = "0.1.0"
 edition = "2024"
 
+[[bin]]
+name = "deploy-queue"
+path = "src/main.rs"
+
+[lib]
+name = "deploy_queue"
+path = "src/main.rs"
+
```

Comment:
This is causing linter errors. I think we should split this into `src/main.rs` for the binary and `src/lib.rs` for the library part. The latter should contain everything that we're testing in our integration tests, while the former should be a thing wrapper around that lib.